pkg/controller/plan/migration.go

```diff
@@ -110,68 +111,6 @@ const (
 	DvStatusCheckRetriesAnnotation = "dvStatusCheckRetries"
 )
 
-var (
```

Comment:
You got leftover predicates `// Predicates.`  which are not used.

Reply:
Thanks